### PR TITLE
bump version to 1.0.0

### PR DESCRIPTION
**Pull request by jyane**

### CODE REVIEW

**Review by ericgribkoff — APPROVED** [review]

**Comment by ericgribkoff** [comment]
@jyane Hi - just wanted to check to make sure you're not waiting on me for anything here? I think once this is merged and the tag created I can do the release

**Comment by jyane** [comment]
Ah sorry, I'm just waiting for:
https://github.com/grpc/grpc-java-api-checker/issues/3#issuecomment-367427415
>  I'll be ready to deploy the package once you update the name.

But it seems to be my misunderstanding...
Merging.

**Comment by jyane** [comment]
@ericgribkoff I've pushed the `v1.0.0`.

**Pull request merged**
